app/src/js/components/Footer/footer.js

```diff
@@ -18,7 +20,10 @@ class Footer extends React.Component {
       <div className='footer'>
         <div className='api__summary'>
           { authenticated &&
-            <h5 className='api__version'>Cumulus API Version: { versionNumber }</h5>
+            <div>
+              <h5 className='dashboard_version'> Cumulus Dashboard Version: { DASH_VERSION } </h5>
```

Comment:
can we update the class with two underscores so it's consistent with `api_version`?
Also, can we update this so that it uses the same styles as `api_version`? 
```suggestion
              <h5 className='dashboard__version'> Cumulus Dashboard Version: { DASH_VERSION } </h5>
```

Basically would need to do this in _base.scss:

```
.api__version,
.dashboard__version{
  font-weight: 400;
  color: $white;
  padding: 10px 0 2px 0;
}
```